tests/osautoinst/worker.pm

```diff
@@ -1,15 +1,16 @@
 use strict;
-use base "openQAcoretest";
+use base 'openQAcoretest';
 use testapi;
-use utils;
+use utils qw(wait_for_desktop switch_to_root_console clear_root_console);
 
 sub run {
     wait_for_desktop;
-    send_key 'ctrl-alt-f3';
+    switch_to_root_console;
     assert_screen 'inst-console';
     type_string "root\n";
-    assert_screen 'password-prompt', 10;
-    type_string $testapi::password . "\n";
+    assert_screen 'password-prompt';
+    type_password;
+    send_key 'ret';
```

Comment:
I guess what we really want here is this non-existing command:

```suggestion
    enter_password;
```

In that sense I find both the old and your new version equally desirable